beancount_reds_importers/importers/etrade/__init__.py

```diff
@@ -9,7 +9,7 @@ class Importer(investments.Importer, ofxreader.Importer):
 
     def custom_init(self):
         self.max_rounding_error = 0.11
-        self.filename_pattern_def = '.*etrade'
+        self.filename_pattern_def = '.*(QFX|qfx)'
```

Comment:
File extensions are [already checked](https://github.com/redstreet/beancount_reds_importers/blob/18cd62c3d7e2b282012dd0e3791d41ce906034f5/beancount_reds_importers/libreader/reader.py#L17). The filename here is just a sensible default for what you can set [bean-download](https://reds-rants.netlify.app/personal-finance/direct-downloads/) to write to, and is designed to be configured via your import config. So it's best to revert this.

Reply:
Thanks for the prompt reply! I had that same thought about naming as well, but I figured the "INV" in the field name stood for Investment and was redundant since this is all for investments, and people might get more confused about what "invexpense" means over "expense", which could be re-used if there are other fields that represent incurred expenses.

Thanks for explaining the filename match- I've reverted that change and updated the unit tests to have a file name that matches.